content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -0,0 +1,168 @@
+
+1. Run the following commands to clone this repository, initialize your `.env` file, create a secure password and create your data directories:
+
+    ```sh
+    cd ~
+    git clone https://github.com/medic/cht-monitoring.git
+    cd cht-monitoring
+    cp cht-instances.example.yml cht-instances.yml
+    cp grafana/grafana.example.ini grafana/grafana.ini
+    mkdir -p grafana/data && mkdir  -p prometheus/data 
+    apt install -y wamerican  # ensures /usr/share/dict/words is present for shuf call below 
+    cp .env.example .env
+    password=$(shuf -n7 /usr/share/dict/words --random-source=/dev/random | tr '\n' '-' | tr -d "'" | cut -d'-' -f1,2,3,4,5,6,7)
+    sed -i -e "s/password/$password/g" .env
+    echo;echo "Initial project structure created! To log into Grafana in the browser:";echo 
+    echo "    username: medic"
+    echo "    password: ${password}";echo
+    ```
+   Note that in step 4 below you'll need the username and password which is printed after you run the above command.
+
+2. Edit the `cht-instances.yml` file to have the URLs of your CHT instances. You may include as many URLs of CHT instances as you like. To preserve data consistency parameters in this file should NOT be adjusted after it is initially set.
+
+   Here is an example:
+
+    ```yml
+    - targets:
+      - https://subsub.sub.example.com
+      - https://cht.domain.com
+      - https://website.org
+    ```
+3. Run the following command to deploy the stack:
+
+    ```sh
+    cd ~/cht-monitoring
+    docker compose up -d
+    ```
+
+4. Grafana is available at [http://localhost:3000](http://localhost:3000). See the output from step 1 for your username and password.
+
+If you would like to do more customizing of your deployment, see ["Addition Configuration"](#additional-configuration).
+
+### Upgrading
+
+Before upgrading, you should back up both your current configuration settings and your Prometheus time-series data.
+
+#### Prometheus, Grafana and JSON Exporter
+
+To upgrade these dependencies, update the version numbers set in your `.env` file (or leave them set to `latest`).  Then run the following commands:
+
+```shell
+docker compose pull
+docker compose up -d --remove-orphans
+```
+
+#### CHT Monitoring Config
+
+When you see a new version in the [GitHub repository](https://github.com/medic/cht-monitoring), first review the release notes and upgrade instructions. Then, run the following commands to deploy the new configuration:
+
+1. Note the version number to derive the branch to pull.  For example,  `1.1.0` would be `1.1.x`
+2. Run this command to update your instance and restart it. Be sure to replace `BRANCH_NAME` with the value from the first step:
+   ```shell
+   cd ~/cht-monitoring
+   git fetch
+   git checkout BRANCH_NAME
+   docker compose pull
+   docker compose down
```

Comment:
```suggestion
```
I don't think we need `down` here.  My understanding is that, `up` is smart enough to re-create any container that has a new image (and just leaves the rest alone.)

Reply:
So - I'm learning some good stuff about longitudinal docker compose health - thank you!

That said, I had `restart` in there originally because we may update something like the `prometheus.ini` or, as I often do the `cht-instances.yml`.  Will `docker compose up` stat those file to see if they've changed such that we can safely remove the `down`?  My (untested!!) assumption was that it wouldn't and the `down` would be needed.

Reply:
> we may update something like the prometheus.ini

Ohh, good point!  Yeah, probably safer to stick with this `down` in that case.  (AFIK, both Prometheus and Grafana do a good job hot-reloading their config files, but I like to be on the safe side)